hack/lib/golang.sh

```diff
@@ -857,3 +857,55 @@ kube::golang::build_binaries() {
     fi
   )
 }
+
+# glog flags
+readonly GLOG_FLAGS=(
```

Comment:
this seems fragile, and also why is it in hack/lib/golang.sh? this file is for code used to build go

Reply:
In my opinion, the `hack/lib/golang.sh` is for code used to build go and run go(in form of `go run`, `go test`). As you can see https://github.com/kubernetes/kubernetes/blob/master/test/integration/ipamperf/test-performance.sh#L73, there `kube::golang::setup_env` is used by `go test`